runtime/src/lib.rs

```diff
@@ -204,57 +203,83 @@ parameter_types! {
 		})
 		.avg_block_initialization(AVERAGE_ON_INITIALIZE_RATIO)
 		.build_or_panic();
-	pub const SS58Prefix: u8 = 42;
+	pub const SS58Prefix: u8 = 0;
```

Comment:
```suggestion
	pub const SS58Prefix: u8 = 42;
```
use the right one ;)